package.json

```diff
@@ -12,14 +13,14 @@
     "lint:md": "markdownlint README.md",
     "lint:es": "eslint --ignore-pattern tests/fixtures/ '{bin,src,tests}/**/*.ts' ",
     "lint:ts": "tsc --noEmit",
-    "dev": "LOG_LEVEL=silly ts-node bin/matrix-appservice-wechaty.ts",
-    "dev:genarate-registration": "ts-node bin/matrix-appservice-wechaty.ts --config config.yaml --url http://localhost:8788 --generate-registration",
-    "dev:service": "ts-node bin/matrix-appservice-wechaty.ts --config config.yaml --file wechaty-registration.yaml",
+    "dev": "LOG_LEVEL=silly node --loader=ts-node/esm bin/matrix-appservice-wechaty.ts",
+    "dev:genarate-registration": "node --loader=ts-node/esm bin/matrix-appservice-wechaty.ts --config wechaty-config.yaml --url http://localhost:8788 --generate-registration",
+    "dev:service": "node --loader=ts-node/esm bin/matrix-appservice-wechaty.ts --config wechaty-config.yaml --file wechaty-registration.yaml",
```

Comment:
Not sure if this is the appropriate solution. Need your further confirmation. @huan

Reply:
> node --loader=ts-node/esm 

This is good.